boot/boot_serial/src/boot_serial.c

```diff
@@ -235,7 +235,8 @@ bs_list(char *buf, int len)
                 flash_area_read(fap, 0, &hdr, sizeof(hdr));
             }
 
-            fih_ret fih_rc = FIH_FAILURE;
+            fih_ret fih_rc;
```

Comment:
The same pattern occurs multiple times. What about FIH_DECLARE?
https://github.com/mcu-tools/mcuboot/pull/1528#discussion_r1039593644

Reply:
You are right, let's implement the `FIH_DECLARE()` macro. I will propose a new commit with this change.